README.md

```diff
@@ -64,15 +68,15 @@ $ evince main.pdf
 # open code/mk_figuresnstats.py with an editor of your choice. Adjust the color palette
 # in the function plot_relationships() from "muted" to "Blues". You can also do
 # this from the terminal with this line of code:
-$ sed -i 's/muted/Blues/g' code/mk_figuresnstats.py  
+$ sed -i'.bak' 's/muted/Blues/g' code/mk_figuresnstats.py  
```

Comment:
I tried this, and it seems ``sed`` works differently on mac and Linux. I suggest doing it like this:
```suggestion
# macOS
$ sed -i'.bak' 's/muted/Blues/g' code/mk_figuresnstats.py  
# linux
$ sed -i 's/muted/Blues/g' code/mk_figuresnstats.py  
```

Reply:
Yes, it produces a backup file on MacOS too, I just don't know how to avoid it. 

I tried using:
`sed -i' ' 's/muted/Blues/g' code/mk_figuresnstats.py`
But it didn't work for me. :(

I +1 to your suggestion, will try to investigate this more to see if I can find a workaround.

Reply:
Could you try
```
sed -i '' s/muted/Blues/g' code/mk_figuresnstats.py
```
?

Reply:
Okay, sorry for the confusion!

This *won't* work on MacOS:
`sed -i'' 's/muted/Blues/g' code/mk_figuresnstats.py`

This *will* (space between quotation marks needed):
`sed -i' ' 's/muted/Blues/g' code/mk_figuresnstats.py`

Reply:
ha. I don't understand why it is different, but good to know! Could you check if its correct in the "suggested change" now?

Reply:
Yes, looks good!